docs/trex_release-notes.md

```diff
@@ -11,6 +11,27 @@ layout: docs
 ----
 See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
+
+### Tableau 2022.3
+
+*October 2022*
+
+* Tableau Dashboard Extensions API library: `tableau.extensions.1.9.0.js` <br>(download or clone the Extensions API repository on [GitHub](https://github.com/tableau/extensions-api){:target="_blank"}.) <br/>
+
+* Download [Tableau Desktop](https://www.tableau.com/support/releases){:target="_blank"} or [Tableau Server](https://www.tableau.com/support/releases/server){:target="_blank"}.
+
+* To preview new features and test your extension with the latest version of Tableau in the Developer Sandbox, join the [Tableau Developer Program](http://www.tableau.com/developer){:target="_blank"} and request your own {{site.tol}} developer site.
+
+About this release:
```

Comment:
We also need to add the pagination examples. Should we focus on this for the viz API first and then have another MR for pagination? Note that when we add pagination examples, we should add the "Certain features in this release are only available in Tableau 2022.4 or later." disclaimer.

Reply:
Right. I was hoping to do doing a quick release now for VizAPI.  
The pagination would be part the next release for 2022.4.

Reply:
Added note about the 2022.4 (or later) requirement to the release notes.